runtime/src/accounts_db.rs

```diff
@@ -413,6 +415,8 @@ pub struct AccountsDB {
     dead_slots: RwLock<HashSet<Slot>>,
 
     stats: AccountsStats,
+
+    operating_mode: Option<OperatingMode>,
```

Comment:
@ryoqun double checking to be extra sure here since there were a lot of merge conflicts 😓. 

Adding this field here should be safe (won't affect serialization of accounts/snapshot generation)?

Reply:
@carllin Yeah, I think this should be safe.